utils/githooks/install

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -eu
+
+githook_root="$(dirname $0)"
+repo_root="${githook_root}/../../"
+
+for d in ${githook_root}/*.d; do
+    h=${d%.d}
+    h=${h##*/}
+    link_name="${repo_root}/.git/hooks/$h"
+    if [ -f "$link_name" ]; then
+        rm "$link_name"
```

Comment:
This is kind of nasty.  It will remove a user's customized git hook without warning even.  Instead it should be moved to the hook's `.d` dir in `utils/githooks/` -- either by this tool or manually by alerting the user and asking them to move it.

Reply:
I opted to warn the user